front/src/dashboard/components/BSDList/BSFF/WorkflowAction/SignTransport.tsx

```diff
@@ -114,7 +116,7 @@ function SignTransportForm({
         onClose();
       }}
     >
-      {() => (
+      {values => (
```

Comment:
```suggestion
      {({values}) => (
```
ou alors 

```suggestion
      {formikProps => (
```

Car sinon en dessous tu fais `values.values`

Reply:
Tout à fait, merci